test/metabase/cmd/compare_h2_dbs.clj

```diff
@@ -112,7 +119,17 @@
 (defn different-contents?
   "Diff contents of 2 DBs. Returns truthy if there is a difference, falsey if not."
   [db-file-1 db-file-2]
-  (jdbc/with-db-connection [conn-1 (jdbc-spec db-file-1)]
-    (jdbc/with-db-connection [conn-2 (jdbc-spec db-file-2)]
-      (or (different-table-names? conn-1 conn-2)
-          (different-rows? conn-1 conn-2)))))
+  (let [spec-1 (jdbc-spec db-file-1)
+        spec-2 (jdbc-spec db-file-2)]
+    (sql-jdbc.execute/do-with-connection-with-options
```

Comment:
Fixes this linting warning:

https://github.com/kienstra/metabase/blob/354f152e113724e50a513a64db118fc14bc8684f/.clj-kondo/config.edn#L134